src/components/page-layout/components/content.js

```diff
@@ -64,9 +64,18 @@ export class ContentWrapper extends React.Component {
 
   renderAside = (showFeedback) => {
     const { frontMatter } = this.props;
-    const { layout } = frontMatter;
+    const { layout, onThisPage } = frontMatter;
     const headings = frontMatter.headings || this.props.headings;
-    const showToc = headings && headings.length > 0 && layout === 'page';
+    let showOnThisPage;
+    // Use `onThisPage` frontmatter to conditionally hide on page layouts or show elsewhere.
+    // By default, show `OnThisPage` only for 'page' layouts.
+    if (typeof onThisPage !== 'undefined') {
+      !onThisPage ? (showOnThisPage = false) : (showOnThisPage = true);
+    } else {
+      layout === 'page' ? (showOnThisPage = true) : (showOnThisPage = false);
+    }
```

Comment:
@danswick I think we can streamline this logic by adding `onThisPage` as a layout option:

1. Add `onThisPage: true` to "page" layout object in `layout.config.js`. (This can replace `sidebar: toc` which is an old configuration.)
2. Then we can update the lines above with similar logic as the layout/frontMatter here to hide feedback: https://github.com/mapbox/dr-ui/blob/5c7c698c23644f75baa0f118bfb0063e9c2a0791/src/components/page-layout/components/content.js#L111-L114

Something like:

```js
// check frontmatter than default to layout config
const showOnThisPage = frontMatter.onThisPage ? frontMatter.onThisPage : layoutConfig.onThisPage;
```